deblur/workflow.py

```diff
@@ -720,13 +720,14 @@ def create_otu_table(output_fp, deblurred_list,
         csampidx = len(sampset)-1
         # read the fasta file and add to the matrix
         for chead, cseq in sequence_generator(cfilename):
+            cseq = cseq.upper()
             if cseq not in seqdict:
                 seqdict[cseq] = len(seqlist)
                 seqlist.append(cseq)
             cseqidx = seqdict[cseq]
             cfreq = float(sizeregexp.search(chead).group(0))
             try:
-                obs[cseqidx, csampidx] = cfreq
+                obs[cseqidx, csampidx] += cfreq
```

Comment:
Not clear to me what's going on here ...

Reply:
Amnon is composing the deblur count table from single counts per sample per feature. Since we might have the situation that we have counts for one sample for feature "ACUGA" with let's say 10 reads and also 5 reads for feature "ACUGa" we need to make sure that those reads are added up to be 15.
Thus, the same obs[cseqidx, csampidx] can be updated several times.